lib/ansible/executor/task_executor.py

```diff
@@ -690,7 +690,7 @@ def _get_connection(self, variables, templar):
 
         if self._play_context.accelerate:
             # accelerate is deprecated as of 2.1...
-            display.deprecated('Accelerated mode is deprecated. Consider using SSH with ControlPersist and pipelining enabled instead')
+            display.deprecated('Accelerated mode is deprecated. Consider using SSH with ControlPersist and pipelining enabled instead', version="2.3", removed=True)
```

Comment:
removed=True is present here but removed has been taken out of the deprecated function.